incident/api/views.py

```diff
@@ -123,6 +125,30 @@ def dispatch(self, *args, **kwargs) -> 'HttpResponse':
 
         return response
 
+    @cached_property
+    def csv_format_is_requested(self):
+        return getattr(self.request, 'accepted_renderer', None) and \
+            self.request.accepted_renderer.format == 'csv'
+
+    @cached_property
+    def requested_fields(self):
+        if fields := self.request.GET.get('fields'):
+            return set(map(str.strip, fields.split(',')))
+        else:
+            return set()
```

Comment:
Small knitpick but I think the `else` is not needed since the `if` block has a return statement.
```suggestion
        if fields := self.request.GET.get('fields'):
            return set(map(str.strip, fields.split(',')))
        return set()
```

Reply:
Good point, I do like this more